frontend/src/metabase/components/form/widgets/FormInputWidget.jsx

```diff
@@ -14,7 +14,7 @@ const propTypes = {
   autoFocus: PropTypes.bool,
   helperText: PropTypes.node,
   tabIndex: PropTypes.string,
-  fieldName: PropTypes.string,
+  subtitle: PropTypes.string,
```

Comment:
`subtitle` is an attempt to keep this more general than `fieldName`.

A better name is always welcome.

Reply:
Good enough for me!